app/controllers/v1/supported_ssgs_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'yaml'
+
+module V1
+  # API for Supported SSGs mapped to RHEL minor versions
+  class SupportedSsgsController < ApplicationController
+    def index
+      render_json scope_search
+    end
+
+    private
+
+    def metadata(opts = {})
+      opts[:total] ||= scope_search.count
+      {
+        meta: {
+          total: opts[:total],
+          revision: resource.revision
+        }
+      }
+    end
```

Comment:
I like the idea of adding the revision in the metadata; it seems appropriate. :+1: It'd be good to get search, pagination, and links working like the other endpoints as well. Pagination isn't really necessary, but I'd push for it just for the sake of consistency.

Reply:
We might introduce search and pagination when we enhance the framework to work with non-db resources. I've played with it a bit, and it did not fit the current state (e.g. the `.count` only counted paginated items ...). It might be a good idea to reconsider the use of a db model with a sync job.